packages/html/src/slider/tests/slider-tick-absolute-position.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { Button } from '../../button';
+
+const root = ReactDOM.createRoot(
+    document.getElementById('app') as HTMLElement
+);
+
+const style = `
+    .k-slider-horizontal {
+        width: 1px;
+        width: 100%;
+    }
+    .k-slider-vertical {
+        height: 300px;
+    }
+
+    .k-slider {
+        --kendo-slider-start: 0;
+        --kendo-slider-end: 60;
+    }
+    .k-range-slider {
+        --kendo-slider-start: 20;
+        --kendo-slider-end: 60;
+    }
+`;
+
+root.render(
+    <>
+        <style>{style}</style>
+        <div id="test-area" className="k-d-grid k-grid-cols-4">
+
+            <section>
+                <div className="k-widget k-slider k-slider-horizontal">
+                    <Button icon="arrow-w" rounded="full" className="k-button-decrease"></Button>
+                    <div className="k-slider-track-wrap">
+                        <ul className="k-reset k-slider-items" style={{ position: 'relative' }}>
+                            <li className="k-tick k-tick-large k-tick-horizontal" style={{ zIndex: 1, position: 'absolute', left: '0%' }}>
+                                <span className="k-label">0</span>
+                            </li>
+
+                            <li className="k-tick k-tick-large k-tick-horizontal" style={{ zIndex: 1, position: 'absolute', left: '33%' }}>
+                                <span className="k-label">33</span>
+                            </li>
+
+                            <li className="k-tick k-tick-large k-tick-horizontal" style={{ zIndex: 1, position: 'absolute', left: '50%' }}>
+                                <span className="k-label">50</span>
+                            </li>
+
+                            <li className="k-tick k-tick-large k-tick-horizontal" style={{ zIndex: 1, position: 'absolute', left: '75%' }}>
+                                <span className="k-label">75</span>
+                            </li>
+
+                            <li className="k-tick k-tick-large k-tick-horizontal" style={{ zIndex: 1, position: 'absolute', left: '100%' }}>
+                                <span className="k-label">100</span>
+                            </li>
+                        </ul>
+                        <div className="k-slider-track">
+                            <div className="k-slider-selection" style={{ width: '33%' }}></div>
+                            <a href="#" className="k-draghandle k-draghandle-end" style={{ left: '33%' }}></a>
```

Comment:
Check whether the drag handle needs z-index as well, cause right now it's displayed under the ticks. It should be over them.